app/shop.py

```diff
@@ -0,0 +1,61 @@
+import math
+
+from typing import Any
+
+
+class Shop:
+    def __init__(self, name: str, location: list, products: dict) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def __str__(self) -> str:
+        return f"{self.name}"
+
+    def costs_for_trip_to_shop(self,
+                               customer: Any,

```

Comment:
```suggestion
                               customer: Customer,

```

Reply:
when I import Customer class to shop module I got ImportError

Reply:
double import in main.py  and  shop.py is imposible